README.rst

```diff
@@ -85,3 +85,22 @@ Forum
 ^^^^^^
 
 https://mne.discourse.group
+
+A Note About Connectivity
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the neuroscience community as of 2021, the term "connectivity" can have many
+different meanings. Some common interpretations are:
+
+- all-to-all values computed from some bivariate or multivariate function over signals
+- "actual" connectivity as a graph, where edge values indicate a "connection", and an edge value of zero indicates "no connection"
+
+In the graph interpretation, this can be further stratified into:
+
+- directed, or undirected graph (symmetric, or asymmetric)
+- weighted, or unweighted graph (values of 0 and 1 vs continuous values)
+
```

Comment:
Mh, I am not sure if your point 1 (all-to-all bivariate/multivariate function) is not the same as point 3 (directed). Or in other words: the distinction of directed and undirected as well as weighted and unweighted is not exclusive to graph measures.

I wonder if we should just subscribe to a published opinion on this, WDYT @larsoner ? 
There is the tutorial paper I mentioned before by _Bastos & Schoffelen 2016_ which have a IMO good taxonomy on connectivity measures.

Reply:
I am not confident in the correct taxonomy, so I agree that having / following some paper's definitions is the way to go

Reply:
Okay added a link to the paper and removed this text. Can save it for a statistics tutorial!